### ISSUE TITLE
Back button pointing to opposite side on RTL direction 

### ISSUE THREAD

**Issue by AsimNet** [driftyco/ionic#10685]
**Ionic version:**  
[ ] **1.x**
[x] **2.x**

**I'm submitting a ...**  
[x] bug report
[ ] feature request
[ ] support request

**Current behavior:**

the arrow points to the left !
![incorrect](https://cloud.githubusercontent.com/assets/3624441/23683747/b47d1b84-03ab-11e7-94b9-ddccb20af89e.JPG)

**Expected behavior:**

the arrow points to the right
![correct](https://cloud.githubusercontent.com/assets/3624441/23683746/b47a9558-03ab-11e7-8d4b-45dfd77917a9.JPG)

**Steps to reproduce:**

make the app direction RTL by: 
1- `<ion-nav #content [root]="rootPage" dir="rtl"></ion-nav>`
2- see the back button on navbar

and the same for iOS

**Comment by AsimNet** [comment]
same issue with alert controller, it's still LEFT TO RIGHT..

thanks

**Comment by jgw96** [comment]
Thanks for using Ionic, we will look into this.

**Comment by AsimNet** [comment]
Hey, thanks for creating it ^_^

**Comment by AmitMY** [comment]
imo, on RTL, all of the icons (or at least all of the arrow icons) should be mirrored.. 
So, counter intuitively, `arrow-forward` would be `arrow-back`, `arrow-dropleft` to be `arrow-dripright` etc..

**Comment by jgw96** [comment]
Hello, thanks for using Ionic! We really appreciate this feedback on RTL support. This is not currently high priority for us at the moment, but we are tracking it in #5035. Because of this I am going to close this issue as a duplicate of that issue and ask that you put your feedback on that issue. Please know that by me closing this issue I am not saying that your feedback is not important, or that RTL support is not important, but I am simply closing this issue as we like to keep our issues as organized as possible so things do not get lost in the weeds (which is better said than done haha).

**Comment by jgw96** [comment]
Hello, after discussion I am actually going to reopen this issue and link it to the issue i linked to above. Sorry for the hassle.

**Comment by AmitMY** [comment]
PR https://github.com/driftyco/ionic/pull/11121 solves this issue, @jgw96 Can you review?

**Comment by AmitMY** [comment]
@AsimNet Fixed in https://github.com/driftyco/ionic/pull/11634
@brandyscarney Can be closed

**Comment by AsimNet** [comment]
please open this issue again, because it returns with V3.5.3

back button isn't flipped to other way with RTL!

**Comment by AmitMY** [comment]
Verified.

**Comment by AmitMY** [comment]
@AsimNet Did you set your `$app-direction` to be `multi` is scss?
The current default is `ltr`, because of a breaking change when setting it to `multi`.
Using multi, I can see the back button correctly.

**Comment by AsimNet** [comment]
I didn't touch $app-direction attribute..
I used platform.setDir('rtl',true);

**Comment by AmitMY** [comment]
In `3.4.0` that is all you had to do. However, because it broke some apps, the default is not `ltr`.
in your `variables.scss` file set `$app-direction: multi` or `$app-direction: rtl` for it to work.

**Comment by AsimNet** [comment]
@AmitMY thanks 🌹
